service/searcher/service.go

```diff
@@ -61,6 +61,12 @@ func New(config Config) (*Service, error) {
 func (c *Service) Search(request Request) (*Response, error) {
 	c.logger.Log("level", "debug", "message", fmt.Sprintf("searching secret for organization %s, ID %s", request.Organization, request.ID))
 
+	// We never expose the credential-default secret. From the outside, this does not exist.
+	if request.ID == "default" {
```

Comment:
I'd like `default` to be a constant || pulled from some configuration.

Reply:
done